packages/cascara/src/ui/Form/tests/Form.test.snap

```diff
@@ -52,6 +55,7 @@ exports[`Form snapshot test 1`] = `
           </span>
           <span
             class="Input"
+            id="Last Name"
```

Comment:
This is a use case I do not think we were thinking about yet. For the readonly version, we should not be adding an id parameter to the span. In this scenario, I believe the correct semantics are to set aria-label on the span with the label name as the value, and the innerhtml being the value itself.

Reply:
The `...spread` operation was removed from the `readonly` scenario in modules. `aria-label` added.